src/containers/home/invest/component.tsx

```diff
@@ -8,12 +8,54 @@ import AnimatedScroll from 'containers/home/animated-scroll';
 import Wrapper from 'containers/wrapper';
 
 import Button from 'components/button';
+import Carousel from 'components/carousel';
 import Icon from 'components/icon';
 
 import CHECK_SVG from 'svgs/ui/check.svg?sprite';
+import CHEVRON_RIGHT from 'svgs/ui/chevron-right.svg?sprite';
 
 const Invest = () => {
   const { data: areasData } = useAreas();
+  const [slide, setSlide] = useState(0);
+
+  const formatedAreasData = useMemo(() => {
```

Comment:
```suggestion
  const CHUNKED_AREAS = useMemo(() => {
```
To keep consistency with the app, all the useMemo are always an uppercase const